wp-cldr.php

```diff
@@ -62,12 +62,15 @@ function wp_cldr_settings() {
 
 	<div class="wrap">
 	<h1><?php esc_html_e( 'WP CLDR settings and info', 'wp-cldr' ); ?></h1>
```

Comment:
I wonder if we shouldn't take out the "settings" bit here (& clarify this is purely informational, at the moment) since there's nothing to set.